pyteamcity/future/artifact.py

```diff
@@ -0,0 +1,93 @@
+import fnmatch
+import os
+
+from .core.utils import parse_date_string
+
+
+class Artifact(object):
+    def __init__(self, build, path=''):
+        self.build = build
+        self.path = path
+        teamcity = self.build.build_query_set.teamcity
+        url = self.build.api_url + '/artifacts/metadata/' + self.path
+        res = teamcity.session.get(url)
+        res.raise_for_status()
+        self._data = res.json()
+
+    @property
+    def name(self):
+        return self._data['name']
+
+    def getsize(self):
+        return self._data.get('size')
```

Comment:
I assume this only works if `self.isfile()` is true? Should we encourage to use the API `Artifact.isfile()` to check if calling `Artifact.getsize()` would raise an error instead of returning `None`?